crates/primitives/src/withdrawal.rs

```diff
@@ -23,8 +22,8 @@ pub struct Withdrawal {
 
 impl Withdrawal {
     /// Return the withdrawal amount in wei.
-    pub fn amount_wei(&self) -> U256 {
-        U256::from(self.amount) * U256::from(GWEI_TO_WEI)
+    pub fn amount_wei(&self) -> u128 {
+        self.amount as u128 * GWEI_TO_WEI as u128
     }
```

Comment:
driveby change

Reply:
Effected with changed inside revm as for increment of balance we take u128, so I needed to change this inside reth.